### ISSUE TITLE
Invalid character in entity name

### ISSUE THREAD

**Issue by dylanvdmerwe** [floatinghotpot/cordova-plugin-nativeaudio#110]
Install latest cordova cli: `npm install -g cordova@latest`

When trying to add this plugin, the following error is displayed:
```
Error parsing /Users/dylan/Desktop/Projects/Cochrane/plugins/cordova-plugin-nativeaudio/plugin.xml.
Error: Invalid character in entity name
Line: 8
Column: 54
Char:
```

Not sure what the problem is?

**Comment by didlich** [comment]
see pull request #109, #94

**Comment by dylanvdmerwe** [comment]
Any chance those pull requests can be merged sometime soon?

**Comment by didlich** [comment]
the PRs are about a half year old, wouldn't count on it
create your own fork or use my
you can execute this
`cordova plugin add https://github.com/didlich/cordova-plugin-nativeaudio#xml-invalid-chars`
in this branch the error is fixed

**Comment by dylanvdmerwe** [comment]
Yeah that's annoying. Thanks for the info.

**Comment by dylanvdmerwe** [comment]
Will leave open until fixed.

**Comment by geshub** [comment]
Yes it would be great to have this error fixed !!

**Comment by didlich** [comment]
@floatinghotpot has merged the PRs related to this error a few hours ago, so you can use this repo again with specific hash since a new version was not released yet

**Comment by floatinghotpot** [comment]
it's merged and published to npm as v3.0.8, thanks to @didlich ~
